### DISCUSSION TITLE
Welcome to SbeamRSS Discussions!

### DISCUSSION THREAD

**Discussion by SuperAIdesu** [sbeam-dev/SbeamRSS#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mahmudandroid** [comment]
am mahmoud abdalla from sudan i am flutter dev , i flowing  your a count to
get new  , i hope to Put my fingerprint to open source like add
notification to sbeam
thanks

‫في الأحد، 13 ديسمبر 2020 في 10:00 ص تمت كتابة ما يلي بواسطة ‪爱酱‬‏ <‪
notifications@github.com‬‏>:‬

**Reply by SuperAIdesu** [reply]
Welcome to the community! You can look at the Todo list at the README page to find out what we're currently working on. If you have experience working with flutter, you can contribute to the project by making pull requests. As for notification feature in this app, I currently don't know how to implement it without cloud integration(GMS etc.). I think push notifications from background process of the app is not quite viable because it drains battery and could easily be killed by the OS. Feel free to leave your idea here. Thanks!

**Comment by gzpaitch** [comment]
Hey @SuperAIdesu, how are you doing?
Hope you're doing great! 

Thanks for this awesome work so far!

I'm new on Flutter at all, and would like to know if is it possible to create a pre-made list with all the RSS links? I don't want to give the option to create/delete other RSS sources, and just to give the news from the RSS sources I've selected. I guess it's simple, but since I'm starting, things get a little longer to get done.

**Reply by SuperAIdesu** [reply]
Hello @gzpaitch !
Thank you for your interest. I assume that you would like to create a fork of this repo to achieve the functionality you have described. Feel free to do that as long as the fork is open source, which is needed to comply with the MPL2.0 license. The main branch will of course give user choices of RSS links.
As for the technical stuff, you will need to replace the `sourcedb` with a predefined data structure. You may need to rewrite some related functions and UIs, but that should be simple. Good luck! :-)
